File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int n = nums.size();
        bool f = false;
        sort(nums.begin(),nums.end());
        for(int i = 0; i< n/2; i++){
            if(nums[i]==nums[i+1] || nums[n-i-1] == nums[n-i-2]){
                f = true;
                break;
            }
        }
        return f;
    }
};

File: Data Structures & Algorithms/longest-common-prefix/submission-0.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        n = len(strs)
        strs.sort()
        m = len(strs[0])
        com = ""
        flag = True
        if m == 0:
            return ""
        for i in range(m):
            for word in strs:
                if word[i]!=strs[0][i]:
                    flag = False
                    break
            if flag == True:
                    com = com + word[i]
            else: break
        return com    

        